static-files/source/static-site/layout.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Layout = ({ children }) => children;
+
+Layout.propTypes = {
+  children: PropTypes.node
```

Comment:
When a new page is added with `@creuna/react-scripts`, children is `undefined` which generates an annoying warning. Having `null` as the default prop for `children` fixes this